### ISSUE TITLE
Feedback page and links

### ISSUE THREAD

**Issue by milen-sage** [Sage-Bionetworks/data_curator#33]
Let's have a link/button to a feedback page containing a text field (along with email link) that allows people to send us a messages (e.g. email) as an initial communication channel.  We can place that link at the bottom right of the page for each tab.

Related to #16 .

**Comment by xdoan** [comment]
update landing page to have a link to the official docs, pop up of the instructions, and links to communication channels.

**Comment by milen-sage** [comment]
Yes, this would be a nice addition! Eventually we can also link to the HTAN standards' definitions when we have the schema visualization/exploration server and app; we started discussing that in the data release/portal meeting so hopefully it will get some traction over the next few weeks.

**Comment by ychae** [comment]
In the data curator app, the feedback button can link to the Jira Service Desk that we've set up for HTAN: https://sagebionetworks.jira.com/servicedesk/customer/portal/1/group/-1
